### PR TITLE
fix to add back the ellipsis

### PR DESCRIPTION
**Pull request by shaneeza**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Style change (purely visual updates; if this is the case, attach a screenshot below!)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

## Feedback Levels

Please try to mark your comments with one of these levels

1. Praise (👏 🚀 🔥 🍾)
2. Opinions, Ideas, Suggestions (💡 👀)
3. Let’s discuss @ Code review (🤷 🤔 🧐 )
4. Doesn’t meet agreed upon standards (😬 ⚠️ ‼️)
5. Missing an edge case (🛑 🙅 ❌)
6. Missing core functionality (⛔️ ☣️ 🚧 🌋 💀)

(Note: Use ⌘-⌃-␣ to open the Emoji keyboard!)

### CODE REVIEW

**Comment by shaneeza** [comment]
Screen Shot 2022-08-03 at 4 20 02 PM

File: packages/tabs/src/TabTitle.tsx [hunk 1]
```diff
@@ -176,9 +173,15 @@ const listTitleStyles = css`
...
-  display: flex;
-  align-items: center;
-  gap: 4px;
+  width: 100%;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+
+  > svg {
+    vertical-align: bottom;
```
Comment (TheSonOfThomp) [inline packages/tabs/src/TabTitle.tsx:182]: Technically `vertical-align: text-bottom;` would be more correct, but this works as well

File: packages/tabs/src/TabTitle.tsx [hunk 2]
```diff
@@ -176,9 +173,15 @@ const listTitleStyles = css`
 `;
 
 const listTitleChildrenStyles = css`
-  display: flex;
-  align-items: center;
-  gap: 4px;
+  width: 100%;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
```
Comment (TheSonOfThomp) [inline packages/tabs/src/TabTitle.tsx:179]: I don't like that we can't use flexbox here, but text overflow doesn't work the way we need to use it

File: packages/tabs/src/TabTitle.tsx [hunk 3]
```diff
@@ -176,9 +173,15 @@ const listTitleStyles = css`
 `;
```
Comment (TheSonOfThomp) [inline packages/tabs/src/TabTitle.tsx:173]: Unrelated to this PR, but I don't remember why that `vertical-align: middle;` declaration isn't just inside the block below

File: packages/tabs/src/TabTitle.tsx (hunk 2 above)
Comment (TheSonOfThomp) [reply packages/tabs/src/TabTitle.tsx:179]: Might be worth adding a comment about why we're not using flex here. Someone (most likely me 🙃) is almost guaranteed to try adding it next time they're in here

**Review by TheSonOfThomp — APPROVED** [review]
Works as written, just noted a potentially clearer CSS property value. LGTM 🦅

File: packages/tabs/src/TabTitle.tsx (hunk 3 above)
Comment (shaneeza) [reply packages/tabs/src/TabTitle.tsx:173]: oh yeah, I'm going to remove it then. It doesn't seem like its needed.

**Pull request merged**